crates/node/core/src/args/rpc_server.rs

```diff
@@ -156,6 +156,10 @@ pub struct RpcServerArgs {
     )]
     pub rpc_gas_cap: u64,
 
+    /// The maximum proof lookback for historical proof generation.
```

Comment:
this does not provide meaningful context.
should mention what this value allows head - window